client/www/pages/docs/storage.md

```diff
@@ -5,131 +5,555 @@ title: Storage
 Instant Storage makes it simple to upload and serve files for your app.
 You can use Storage to store images, videos, documents, and any other file type.
 
-{% callout %}
+## Storage quick start
 
-Storage is still in **beta**, but you can request access [here](https://docs.google.com/forms/d/e/1FAIpQLSdzInffrNrsYaamtH_BUe917EOpcOq2k8RWcGM19XepJR6ivQ/viewform?usp=sf_link)!
+Here's a full example of how to upload and display a grid of images
```

Comment:
Nit: perhaps we add an intro sentence: 

"Let's start with a fresh NextJS app:" or something like this?